example/cumulus-tf/main.tf

```diff
@@ -3,11 +3,18 @@ provider "aws" {
   profile = var.aws_profile
 }
 
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
 data "terraform_remote_state" "data_persistence" {
   backend = "s3"
   config  = var.data_persistence_remote_state_config
 }
 
+data "aws_lambda_function" "sts_credentials" {
+  function_name = "gsfc-ngap-sh-s3-sts-get-keys"
```

Comment:
It's supposed to be, but that's for an NGAP account. In theory we should be able to use a non-ngap account. Why isn't this with the distribution module?

Reply:
The `distribution` module takes an input variable called `sts_credentials_lambda_function_arn`. That module is invoked as part of the `cumulus` module so, to make it configurable there as well, the `cumulus` module has an input variable called `sts_credentials_lambda_function_arn`. This file has it hardcoded because the `cumulus-tf` module (which includes this file) is only used for our integration tests. To the best of my knowledge, it should be a safe assumption that those will only be run in an NGAP account.

Reply:
👍